### DISCUSSION TITLE
Is the API down?

### DISCUSSION THREAD

**Discussion by Lakrix** [isFakeAccount/psnawp#78] [Q&A] [answered]
Getting HTTP Status 429 – Too Many Requests when trying to get a session cookie.

**Comment by isFakeAccount** [comment, marked as answer]
> Getting HTTP Status 429 – Too Many Requests when trying to get a session cookie.

Log out and log-in again and get new npsso. It happens sometimes. Also, make sure to rate limit yourself when using API.
